src/js/webrequest.js

```diff
@@ -775,6 +783,16 @@ function dispatcher(request, sender, sendResponse) {
 
     sendResponse();
 
+  } else if (request.type == "saveErrorText") {
+    let activeTab = badger.tabData[request.tabId];
+    activeTab.errorText = request.errorText;
+
+    sendResponse();
```

Comment:
We can remove the `sendResponse` call here since we don't do anything upon response back in the popup.